### ISSUE TITLE
The dqn_agent.py loss function is the wrong way around

### ISSUE THREAD

**Issue by i1Cps** [philtabor/Deep-Q-Learning-Paper-To-Code#19]
In dqn_agent.py, shouldnt the loss function be:

```       
loss = self.q_eval.loss(q_pred, q_target).to(self.q_eval.device)
```
Instead of:

```        
loss = self.q_eval.loss(q_target, q_pred).to(self.q_eval.device)
```

[Pytorch documentation for reference](https://pytorch.org/docs/stable/generated/torch.nn.MSELoss.html)

**Comment by i1Cps** [comment]
"The magnitude of the loss is always the same regardless of the order."
"The gradients of the loss function, with respect to the weights of the network, for a constant (i.e. the target) is always zero."
